apps/frontend/src/views/Login.vue

```diff
@@ -39,16 +39,60 @@
                     @keyup.enter="login"
                     @blur="$v.password.$touch()"
                   />
-                  <v-btn
```

Comment:
Could this be updated to look as follows?

![login](https://user-images.githubusercontent.com/2308869/104476569-639eca80-558e-11eb-96b6-d263e245aad2.JPG)